.github/actions/next-stats-action/src/run/collect-diffs.js

```diff
@@ -32,7 +32,11 @@ module.exports = async function collectDiffs(
 
       await Promise.all(
         globs.map(async (pattern) => {
-          curFiles.push(...(await glob(pattern, { cwd: statsAppDir })))
+          curFiles.push(
+            ...(await glob(pattern, { cwd: statsAppDir })).filter((file) =>
+              file.endsWith('.js')
```

Comment:
Perhaps the .js should be added to the glob pattern instead?

Reply:
Good point, updated there